wire-ios-data-model/Source/Public/ZMMessage.h

```diff
@@ -77,8 +77,6 @@ typedef NS_CLOSED_ENUM(int16_t, ZMSystemMessageType) {
     ZMSystemMessageTypeDecryptionFailed,
     ZMSystemMessageTypeDecryptionFailed_RemoteIdentityChanged,
     ZMSystemMessageTypeNewConversation,
-    ZMSystemMessageTypeReactivatedDevice __deprecated_enum_msg("Devices can't be reactivated any longer"),
-    ZMSystemMessageTypeUsingNewDevice __deprecated_enum_msg("We don't need inform users about new devices any longer"),
```

Comment:
I just asked myself if we can remove the cases without side effects. For example if the `int16` is stored and loaded, could it have a wrong value then?

Reply:
No we can't remove system message types like this it would change the meaning any persisted system with a higher value than 14

Reply:
Either we keep the deprecated enum values or we start assigning explicit values to the enum cases.

Reply:
Thanks, I would tend to set explicit values then. Do you see any arguments against it?

Reply:
@typfel I've reverted the case removal, please take a look.